.gitignore

```
@@ -47,3 +47,7 @@ coverage.xml
 
 # virtualenv
 ENV/
+
+# Sphinx documentation
+docs/_build/
+docs/stubs/*
```

I don't think the trailing `*` has any effect.  It seems that approxiately half the qiskit repos have it and approximately half do not.
```suggestion
docs/stubs/
```